doc/source/ray-core/api/scheduling.rst

```diff
@@ -10,6 +10,10 @@ Scheduling Strategy
 
     ray.util.scheduling_strategies.PlacementGroupSchedulingStrategy
     ray.util.scheduling_strategies.NodeAffinitySchedulingStrategy
+    ray.util.scheduling_strategies.DoesNotExist
+    ray.util.scheduling_strategies.NotIn
+    ray.util.scheduling_strategies.Exists
+    ray.util.scheduling_strategies.NodeLabelSchedulingStrategy
```

Comment:
exclude those for now since label scheduling is not completed yet and we won't want users to use it yet